src/Patches.php

```diff
@@ -295,7 +295,7 @@ public function postInstall(PackageEvent $event) {
       }
       catch (\Exception $e) {
         $this->io->write('   <error>Could not apply patch! Skipping. The error was: ' . $e->getMessage() . '</error>');
-        if (getenv('COMPOSER_EXIT_ON_PATCH_FAILURE') || !empty($extra['composer-exit-on-patch-failure'])) {
+        if (getenv('COMPOSER_EXIT_ON_PATCH_FAILURE') || !empty($extra['composer-exit-on-patch-failure']) || !empty($this->composer->getConfig()->get('composer-exit-on-patch-failure'))) {
```

Comment:
Looking at this, it seems like maybe the 'extra' line indicates you can configure this in composer.json already in the extra section already?

Reply:
I think you're right--I'd mis-read this and assumed it had to be specified per-package.

Reply:
Yep, @grayside is right. Closing, but if that option doesn't work for you, comment here about how it could be better and we'll reopen.